backend/typescript/services/implementations/cronService.ts

```diff
@@ -0,0 +1,159 @@
+import dayjs from "dayjs";
+import customParseFormat from "dayjs/plugin/customParseFormat";
+import { Op } from "sequelize";
+import IEmailService from "../interfaces/emailService";
+import { UserDonorDTO } from "../../types";
+import logger from "../../utilities/logger";
+import Schedule from "../../models/scheduling.model";
+import User from "../../models/user.model";
+import getErrorMessage from "../../utilities/errorMessageUtil";
+import ICronService from "../interfaces/cronService";
+import IDonorService from "../interfaces/donorService";
+
+// eslint-disable-next-line
+const cron = require("node-cron");
+
+const Logger = logger(__filename);
+
+class CronService implements ICronService {
+  emailService: IEmailService | null;
+
+  donorService: IDonorService;
+
+  constructor(
+    emailService: IEmailService | null = null,
+    donorService: IDonorService,
+  ) {
+    this.emailService = emailService;
+    this.donorService = donorService;
+  }
+
+  async sendScheduledDonationEmail(schedule: Schedule): Promise<void> {
+    if (!this.emailService) {
+      const errorMessage =
+        "Attempted to send email regarding 24 hour email reminder but this instance of SchedulingService does not have an EmailService instance";
+      Logger.error(errorMessage);
+      throw new Error(errorMessage);
+    }
+
+    const currDonor: UserDonorDTO = await this.donorService.getDonorById(
+      String(schedule.donor_id),
+    );
+    // Proposed drop off info
+
+    try {
+      const { firstName, email } = currDonor;
+      const startTime = schedule.start_time;
+      const endTime = schedule.end_time;
+
+      const startTimeToLocalDate = startTime.toLocaleString("en-US", {
+        timeZone: "EST",
+      });
+
+      const startDayString: string = dayjs(startTimeToLocalDate).format(
+        "dddd, MMMM D",
+      );
+
+      const startTimeString: string = dayjs(startTimeToLocalDate).format(
+        "h:mm A",
+      );
+      const endTimeString: string = dayjs(
+        endTime.toLocaleString("en-US", {
+          timeZone: "EST",
+        }),
+      ).format("h:mm A");
+
+      dayjs.extend(customParseFormat);
+
+      const volunteerTimeString =
+        dayjs(schedule.volunteer_time, "HH:mm").format("h:mm A") ?? "";
+
+      const emailBody = `
+        <html>
+          <head>
+            <link
+              href="https://fonts.googleapis.com/css2?family=Inter"
+              rel="stylesheet"
+            />
+            <style>
+              body {
+                  font-family: "Inter";
+              }
+            </style>
+            <meta charset="utf-8" />
+            <meta http-equiv="x-ua-compatible" content="ie=edge" />
+            <title>Donation Details Email</title>
+          </head>
+          <body>
+            <p><img src=https://i.ibb.co/txCj8db/drawer-logo.png
```

Comment:
i believe the image source should be `https://community-fridge-logo.s3.us-west-004.backblazeb2.com/community-fridge-logo.png`